### PR TITLE
Use proposal template & Move syntax definition to emu file (No spec changes)

### PR DESCRIPTION
**Pull request by Jack-Works**

### CODE REVIEW

File: CORE.md [hunk 1]
```diff
@@ -10,26 +10,26 @@
...
-  * [Syntax](#case-xp-syntax)
-* [Annex A: Design Decisions](#annex-a)
-  * [No Clause Fallthrough](#no-fallthrough)
-  * [Variables Always Assign](#variables-always-assign)
-  * [`Object.is` for non-collection literals](#object-is-comparison)
-  * [Only one case param](#only-one-param)
-* [Annex B: Performance Considerations](#annex-b)
-* [Annex C: Future Bikeshedding Concerns](#annex-c)
-  * [C.1 `undefined` and Other "Literals"](#fake-literals)
+-   [Introduction](#introduction)
```
Comment (ljharb) [inline CORE.md:13]: let's revert all these unrelated formatting changes (please be sure your editor isn't set up to blindly run prettier on things)

**Comment by Jack-Works** [comment]
rebased
